src/ogdc_runner/__main__.py

```diff
@@ -1,44 +1,40 @@
 from __future__ import annotations
 
-import time
-
 import click
 
 from ogdc_runner.argo import get_workflow_status, submit_workflow
 from ogdc_runner.recipe.simple import make_simple_workflow
 
-recipe_path = click.argument(
-    "recipe_path",
-    required=True,
-    metavar="RECIPE-PATH",
-    type=str,
-)
-
 
 @click.group
 def cli() -> None:
     """A tool for submitting data transformation recipes to OGDC for execution."""
 
 
-def _submit_workflow(recipe_path: str) -> str:
-    workflow = make_simple_workflow(
-        recipe_dir=recipe_path,
-    )
-    workflow_name = submit_workflow(workflow)
-    print(f"Successfully submitted recipe with workflow name {workflow_name}")
-    return workflow_name
-
```

Comment:
thanks for doing this I think this refactor is a great call at this point and clears up some redundancy